.github/build/Containerfile

```diff
@@ -5,10 +5,12 @@ LABEL summary="Toolchain for running pre-commit hooks." \
     io.k8s.display-name="Pre-Commit Toolchain"
 
 USER root
-RUN dnf install nodejs
+RUN dnf install nodejs -y && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
```

Comment:
Need to install the openshift client cli as it is used directly by the  `openshift-client` python library